samples/Azure.Management.Storage/Generated/StorageAccountCreateParametersContainer.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace Azure.Management.Storage
 {
-    /// <summary> A class representing collection of StorageAccounts and their operations over a [ParentResource]. </summary>
-    public partial class StorageAccountsContainer
+    /// <summary> A class representing collection of StorageAccountCreateParameters and their operations over a [ParentResource]. </summary>
```

Comment:
Why did this become StorageAccountCreateParametersContainer instead of StorageAccountContainer

Reply:
Same reason as above: https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json#L145 I think we should override this value cuz this name looks not reasonable.